src/main/scala/mockws/MockWSHelpers.scala

```diff
@@ -20,22 +20,27 @@ import scala.concurrent.duration.Duration
   * Example:
```

Comment:
I would add a BIG FAT warning that users have to call `shutdownHelpers()` to avoid memory leaks